### PR TITLE
Chore/claim gas tests

### PR DESCRIPTION
**Pull request by mhuggins**
## Description
This adds component & container tests for the `claimGas` API call.

## Motivation and Context
We need more confidence around our API functions to ensure they're doing what's expected.

## How Has This Been Tested?
`yarn test`

## Types of changes
- [x] Chore (tests, refactors, and fixes)
- [ ] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] I have read the **CONTRIBUTING** document.

## Closing issues
N/A

### CODE REVIEW

File: src/renderer/browser/components/RequestsProcessor/Invoke/Invoke.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { string, func } from 'prop-types';
 
-export default class Invoke extends React.Component {
+export default class Send extends React.Component {
```
Comment (DalderupMaurice) [inline src/renderer/browser/components/RequestsProcessor/Invoke/Invoke.js]: Why the need to rename it to `Send`?

Comment (mhuggins) [reply src/renderer/browser/components/RequestsProcessor/Invoke/Invoke.js]: Err, this was a mistake. 😅

Comment (mhuggins) [reply src/renderer/browser/components/RequestsProcessor/Invoke/Invoke.js]: @DalderupMaurice Fixed.  I meant to change the name of the class in `Send.js`, as that was called `Invoke`.

File: src/renderer/browser/components/RequestsProcessor/ClaimGas/index.js [hunk 2]
```diff
@@ -28,11 +29,7 @@ export default function makeClaimComponent(claimActions) {
     withData(authActions, mapAuthDataToProps),
 
     // Do invoke if user accepts
-    withCall(claimActions, ({ net, address, wif }) => ({
-      net,
-      address,
-      wif
-    })),
+    withInitialCall(claimActions, ({ net, address, wif }) => ({ net, address, wif })),
```
[the remark this answers is not in the document]
Comment (mhuggins) [reply src/renderer/browser/components/RequestsProcessor/ClaimGas/index.js]: Yeah, I'm planning to long term.  I just haven't done it yet. 😄

[the remark this answers is not in the document]
Comment (DalderupMaurice) [reply src/renderer/browser/components/RequestsProcessor/ClaimGas/index.js]: Could we have something either in the client/github or spunky to track this? 😄

[the remark this answers is not in the document]
Comment (mhuggins) [reply src/renderer/browser/components/RequestsProcessor/ClaimGas/index.js]: Done: #265.

**Pull request merged**